.github/workflows/build.yml

```diff
@@ -37,3 +37,16 @@ jobs:
             - run: yarn
 
             - run: yarn test:unit
+
+    e2e:
+        runs-on: ubuntu-latest
+        steps:
+            -   uses: actions/setup-node@v1
+                with:
+                    node-version: 14
```

Comment:
should run unit and e2e tests on Node version : v14, v16 , so need changes in `e2e` and `unit` sections

Reply:
done